fdbserver/StorageMetrics.actor.h

```diff
@@ -184,19 +184,23 @@ struct TransientStorageMetricSample : StorageMetricSample {
 struct StorageServerMetrics {
 	KeyRangeMap< vector< PromiseStream< StorageMetrics > > > waitMetricsMap;
 	StorageMetricSample byteSample;
-	TransientStorageMetricSample iopsSample, bandwidthSample;	// FIXME: iops and bandwidth calculations are not effectively tested, since they aren't currently used by data distribution
+	TransientStorageMetricSample iopsSample, bandwidthSample,
+	    bytesReadSample; // FIXME: iops and bandwidth calculations are not effectively tested, since they aren't
```

Comment:
It's better to start `bytesReadSample` in a new line as `TransientStorageMetricSample  bytesReadSample ` since the comment is for the above two metrics.

Reply:
Will do.